.github/workflows/build.yml

```
@@ -0,0 +1,74 @@
+
+      - name: Comment Building
+        uses: thollander/actions-comment-pull-request@v3
+        with:
+          message: |
+            :monkey: Monkeys are building your code...
+          comment-tag: pr_build_message
+          mode: upsert
+
+      - name: setup env
+        id: node-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:2:41: Quote this to prevent word splitting

(shellcheck)